assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-checkout-button-block/block.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "woocommerce/mini-cart-checkout-button-block",
+	"version": "1.0.0",
+	"title": "Mini Cart Checkout Button",
```

Comment:
I don't have any strong opinion, but the _Proceed to Checkout_ button of the Cart block is called "Proceed to Checkout":

![imatge](https://user-images.githubusercontent.com/3616980/226351473-7a9cf4ed-fdfa-40e7-b979-2b96b4ea3135.png)

What about naming the Mini Cart button as "Mini Cart Proceed to Checkout" or "Mini Cart Proceed to Checkout Button" to keep it consistent?